coreflow/src/main/java/id/co/veritrans/sdk/coreflow/core/MerchantRestAPI.java

```diff
@@ -215,5 +217,16 @@ void registerCard(@Header("x-auth") String auth,
     @POST("/charge")
     void getSnapToken(@Body SnapTokenRequestModel requestModel, Callback<Token> callback);
 
+    /*
```

Comment:
JavaDoc: Please use correct template by adding the parameters too in the javadoc.